Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -834,9 +883,31 @@ def upload_file_command(client: Client, args: Dict[str, str]):
 
     if res.get('RequestedObject') and res.get('IsSuccessful'):
         attachment_id = res['RequestedObject'].get('Id')
-        return_outputs(f'File uploaded succsessfully, attachment ID: {attachment_id}', {}, res)
     else:
-        return_error('Upload file failed')
+        raise DemistoException('Upload file failed')
+
+    return_outputs(f'File uploaded succsessfully, attachment ID: {attachment_id}', {}, res)
+    return attachment_id
+
+
+def upload_and_associate_command(client: Client, args: Dict[str, str]):
```

Comment:
![image](https://user-images.githubusercontent.com/11165655/100547241-517efc00-326e-11eb-9842-742157e9f2e9.png)
Can't add it to the test playbook as it will make a lot of garbage in Archer (we can't delete the file).